openstates/ut/bills.py

```diff
@@ -190,57 +200,60 @@ def parse_status(self, bill, status_table):
 
             if action == 'Governor Signed':
                 actor = 'executive'
-                type = 'governor:signed'
+                typ = 'executive-signature'
             elif action == 'Governor Vetoed':
                 actor = 'executive'
-                type = 'governor:vetoed'
+                typ = 'executive-veto'
             elif action.startswith('1st reading'):
-                type = ['bill:introduced', 'bill:reading:1']
+                typ = ['introduction', 'reading-1']
             elif action == 'to Governor':
-                type = 'governor:received'
+                typ = 'executive-receipt'
             elif action == 'passed 3rd reading':
-                type = 'bill:passed'
+                typ = 'passage'
             elif action.startswith('passed 2nd & 3rd readings'):
-                type = 'bill:passed'
+                typ = 'passage'
             elif action == 'to standing committee':
                 comm = row.xpath("td[3]/font/text()")[0]
                 action = "to " + comm
-                type = 'committee:referred'
+                typ = 'referral-committee'
             elif action.startswith('2nd reading'):
-                type = 'bill:reading:2'
+                typ = 'reading-2'
             elif action.startswith('3rd reading'):
-                type = 'bill:reading:3'
+                typ = 'reading-3'
             elif action == 'failed':
-                type = 'bill:failed'
+                typ = 'failure'
             elif action.startswith('2nd & 3rd readings'):
-                type = ['bill:reading:2', 'bill:reading:3']
+                typ = ['reading-2', 'reading-3']
             elif action == 'passed 2nd reading':
-                type = 'bill:reading:2'
+                typ = 'reading-2'
             elif 'Comm - Favorable Recommendation' in action:
-                type = 'committee:passed:favorable'
+                typ = 'committee-passage-favorable'
             elif action == 'committee report favorable':
-                type = 'committee:passed:favorable'
+                typ = 'committee-passage-favorable'
             else:
-                type = 'other'
-            bill.add_action(actor, action, date, type=type,
-                            _vote_id=uniqid)
+                typ = None
+            bill.add_action(description=action,
```

Comment:
I'm thinking it would make sense to add `vote_id` as part of `extras` instead:

```python
action = bill.add_action(...)
action.extras = {'_vote_id': str(uniqid)}
```